wire-tests/src/test/java/com/squareup/wire/java/KotlinGeneratorTest.kt

```diff
@@ -0,0 +1,97 @@
+package com.squareup.wire.java
+
+import com.squareup.kotlinpoet.FileSpec
+import com.squareup.wire.kotlin.KotlinGenerator
+import com.squareup.wire.schema.RepoBuilder
+import junit.framework.TestCase.assertTrue
+import org.junit.Test
+
+class KotlinGeneratorTest {
+  @Test fun basic() {
+    val repoBuilder = RepoBuilder()
+      .add("message.proto", """
+        |message Person {
+        |	required string name = 1;
+        |	required int32 id = 2;
+        |	optional string email = 3;
+        |	enum PhoneType {
+        |		HOME = 0;
+        |		WORK = 1;
+        |		MOBILE = 2;
+        |	}
+        |	message PhoneNumber {
+        |		required string number = 1;
+        |		optional PhoneType type = 2 [default = HOME];
+        |	}
+        |	repeated PhoneNumber phone = 4;
+        |}""".trimMargin())
+    val code = repoBuilder.generateKotlin("Person")
+    assertTrue(code.contains("data class Person @JvmOverloads constructor"))
+    assertTrue(code.contains("object ADAPTER : ProtoAdapter<Person>(FieldEncoding.LENGTH_DELIMITED, " +
+        "Person::class.java) {"))
+    assertTrue(code.contains("override fun encode(writer: ProtoWriter, value: Person)"))
+    assertTrue(code.contains("enum class PhoneType(private val value: Int) : WireEnum"))
+    assertTrue(code.contains("WORK(1),"))
+  }
+
+  @Test fun defaultValues() {
+    val repoBuilder = RepoBuilder()
+        .add("message.proto", """
+            |message Message {
```

Comment:
nit: indentation of multiline strings is inconsistent, some tests use 2-space, some use 4-space